File: Data Structures & Algorithms/two-integer-sum/submission-1.cs
public class Solution {
    public int[] TwoSum(int[] nums, int target) {
        for (int i = 0; i < nums.Length; i++)
        {
            for (int j = 1; j < nums.Length; j++)
            {
                if (i != j && nums[i] + nums[j] == target)
                    return [i, j];
            }
        }

        return [];
    }
}

File: Data Structures & Algorithms/sliding-window-maximum/submission-1.py
class Solution:
    def maxSlidingWindow(self, nums: List[int], k: int) -> List[int]:
            if len(nums) == 0: return []
            if len(nums) == 1: return [nums[0]]
            if k == 1: return nums
            res = []
            for i, n in enumerate(nums):
                rp = i + k
                if rp <= len(nums):
                    res.append(max(nums[i:rp]))
                if rp == len(nums):
                    break
            return res


